--- cppo-1.5.0/Makefile~	2017-04-24 22:34:11.000000000 +0200
+++ cppo-1.5.0/Makefile	2017-06-05 21:47:06.476440992 +0200
@@ -45,9 +45,14 @@
 endif
 endif
 
+ifeq "${BEST}" ".native"
 OCAMLBUILD_INSTALL = ocamlbuild_plugin/_build/ocamlbuild_cppo.cmi \
                      ocamlbuild_plugin/_build/ocamlbuild_cppo.cmx \
                      $(addprefix ocamlbuild_plugin/_build/,$(OCAMLBUILD_IMPL))
+else
+OCAMLBUILD_INSTALL = ocamlbuild_plugin/_build/ocamlbuild_cppo.cmi \
+                     $(addprefix ocamlbuild_plugin/_build/,$(OCAMLBUILD_IMPL))
+endif
 
 
 .PHONY: default all opt toplib install clean test
